### ISSUE TITLE
cannot Build at centos 8

### ISSUE THREAD

**Issue by sp4148** [codayblue/libmyemail#4]
cannot Build at centos 8

with below error 

 fatal error: Poco/Net/SecureSMTPClientSession.h: No such file or directory
 #include "Poco/Net/SecureSMTPClientSession.h" /* Send the email over TLS this is needed */
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/myemail.dir/build.make:63: CMakeFiles/myemail.dir/myemail.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:68: CMakeFiles/myemail.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

**Comment by codayblue** [comment]
Looks like you are missing poco-devel package. I will look into how to get those and add steps to read me. If you beat me too it I will happily accept a PR with corrected steps.

**Comment by sp4148** [comment]
but centos 8 has no this package poco-devel

**Comment by codayblue** [comment]
Hummm yeah that is an issue. You will likely need to compile poco from source. I was able to compile it by downloading the tar.gz from https://github.com/pocoproject/poco/releases/tag/poco-1.10.1-release then untar it and change into the directory. From there I ran the commands below:
```
sudo yum install -y git gcc-c++ make cmake3 openssl-devel mariadb-devel
./configure
make
sudo make install
```
After that I tried compiling this lib again and it compiled fine. I will install a test mysql and test the lib when I get a chance.
